fvm/src/kernel/default.rs

```diff
@@ -823,9 +823,11 @@ where
         actor_id: ActorID,
         delegated_address: Option<Address>,
     ) -> Result<()> {
-        let is_allowed_to_create_actor = self.actor_id == INIT_ACTOR_ID
-            || (cfg!(feature = "testing")
-                && self.actor_id == TEST_ACTOR_ALLOWED_TO_CALL_CREATE_ACTOR);
+        let is_allowed_to_create_actor = self.actor_id == INIT_ACTOR_ID;
+
+        #[cfg(feature = "testing")]
+        let is_allowed_to_create_actor =
+            is_allowed_to_create_actor || self.actor_id == TEST_ACTOR_ALLOWED_TO_CALL_CREATE_ACTOR;
```

Comment:
This is an idiom I use when using feature flags. Re-using the variable is painful because:

1. When the feature is enabled, the binding needs to be mutable.
2. When disabled, the binding needs to be immutable (you'll get a warning).

Simply re-binding the variable like this avoids that situation.

Reply:
Good to know, thanks!